### PR TITLE
finish etron second-block

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/js/templates/SecondBlock.js [hunk 1]
```diff
@@ -18,17 +21,19 @@ bodyColorControls.forEach((element, index) => {
...
-        secondBlockVideo.setAttribute('poster', `css/images/second-block-${secondBlockIndexes.firstIndex}${secondBlockIndexes.secondIndex}.jpg`)
+        secondBlockVideo.setAttribute('src', `videos/${carModel}_${secondBlockIndexes.firstIndex}${secondBlockIndexes.secondIndex}.mp4`)
+        secondBlockVideo.setAttribute('poster', `css/images/${carModel}_second-block-${secondBlockIndexes.firstIndex}${secondBlockIndexes.secondIndex}.jpg`)
     })
 })
 
 wheelsControls.forEach((element, index) => {
     element.addEventListener('click', () => {
-        toggleActiveClass(element, wheels)
+        console.log(element)
```
Comment (StadniukYevhen) [inline src/js/templates/SecondBlock.js]: remove `console.log`

File: src/js/templates/SecondBlock.js [hunk 2]
```diff
@@ -18,17 +21,19 @@ bodyColorControls.forEach((element, index) => {
...
+        console.log(wheels)
```
Comment (StadniukYevhen) [inline src/js/templates/SecondBlock.js]: remove `console.log`

File: src/js/handlers/classHandlers.js [hunk 3]
```diff
@@ -98,6 +98,11 @@ const toggleActiveClass = (newElement, wrapper) => {
     toggleActive(previousActiveElement, newElement)
 }
 
+const toggleWheelsActiveClass = (newElement, wrapperClass) => {
+    const previousActiveElement = document.querySelector(`.${wrapperClass} .active`)
```
Comment (StadniukYevhen) [inline src/js/handlers/classHandlers.js]: it's unnecessary to create variable

Comment (nmelnychenko) [reply src/js/handlers/classHandlers.js]: remove function

File: src/js/templates/SecondBlock.js (hunk 1 above)
Comment (nmelnychenko) [reply src/js/templates/SecondBlock.js]: done

File: src/js/templates/SecondBlock.js (hunk 2 above)
Comment (nmelnychenko) [reply src/js/templates/SecondBlock.js]: done

**Pull request merged**